src/components/explorer/builder/QueryPickerContainer.js

```diff
@@ -24,11 +36,23 @@ const localMessages = {
   querySearch: { id: 'explorer.queryBuilder.advanced', defaultMessage: 'Search' },
   searchHint: { id: 'explorer.queryBuilder.hint', defaultMessage: 'Search' },
   deleteFailed: { id: 'explorer.queryBuilder.hint', defaultMessage: 'Sorry, deleting your search failed for some reason.' },
+  quickCompareTitle: { id: 'explorer.queryBuilder.quickCompareTitle', defaultMessage: 'Replace Your Queries?' },
+  quickComparePartisanship: { id: 'explorer.queryBuilder.quickCompareText.partisanship',
+    defaultMessage: 'This will replace your queries with a set that lets you compare coverage in media sources across the US political spectrum. Your search terms and dates will be kept, but we will create 5 quries - one each for the left, center left, center, center-right, and right.' },
```

Comment:
A few nits:
- "US" should be "U.S."
- misspelling -- "quries"
- "center left" needs a hyphen too